ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -286,7 +286,16 @@ - (void)_rebuildBridge
       [self _enforceDesiredDeviceOrientation];
       [self _invalidateRecoveryTimer];
       [[EXKernel sharedInstance] logAnalyticsEvent:@"LOAD_EXPERIENCE" forAppRecord:self.appRecord];
-      [self.appRecord.appManager rebuildBridge];
+
+      @try {
+        [self.appRecord.appManager rebuildBridge];
+      }
+      @catch (NSException *exception) {
+        // Catch errors related to invalid manifests, specifically if the manifest doesn't have a scopeKey.
+        [self maybeShowError:[NSError errorWithDomain:@"ExpoRebuildingBridge"
+                                                 code:1023
+                                             userInfo:@{NSLocalizedDescriptionKey: [@"Failed to rebuild the bridge: " stringByAppendingString:exception.reason] }]];
```

Comment:
might be good to pass exception through if possible so that maybe we can rethrow that instead of constructing a new NSException in standalone apps